src/Misc/layoutbin/en-US/strings.json

```diff
@@ -245,6 +246,13 @@
     "--username     basic authentication username",
     "--password     basic authentication password",
     "",
+    "",
+    "Arguments for enableautologon (Only on Windows):",
+    "-----------------------------------------------------------------------------",
+    "--restartnow          Used after autologon configuration is done. Recommended to restart the machine for the configured settings to take place.",
```

Comment:
what's the default value for restartnow/disablescreensaver?
base on the code both default is true, which is wrong. how can i set it to false in unattended mode?
please fix them.